core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/visualization/barChart/BarChartOpDesc.scala

```diff
@@ -21,7 +21,7 @@ import edu.uci.ics.texera.workflow.operators.visualization.{
   VisualizationOperator
 }
 
-//type constraint: value can only be numeric
+ //type constraint: value can only be numeric
```

Comment:
Please avoid changing unrelated code as much as possible in a PR.

Reply:
I removed that extra space.